Bump configparser dependency back to 3 again.

Chapter 1:

Upstream lowered this dependency (for the benefit of Fedora?) as part of
https://github.com/AsahiLinux/speakersafetyd/commit/063db4854bb02927a1b24b8bf3577f51b2c305a4#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542L11

As debian has the newer configparser crate packaged, upgrade it again.


Chapter 2:

Upstream now bumped to 3.0.3, but debian only has 3.0.2 still, so lower it for now.


Index: speakersafetyd/Cargo.toml
===================================================================
--- speakersafetyd.orig/Cargo.toml
+++ speakersafetyd/Cargo.toml
@@ -32,7 +32,7 @@ features = ["derive"]
 version = "2.0.0"
 
 [dependencies.configparser]
-version = "^3.0.3"
+version = "^3.0.2"
 features = ["indexmap"]
 
 [dependencies.json]
